generator/README.md

```diff
@@ -10,17 +10,30 @@ Updating the generated code can be done only by modifying the code generator, or
 To run the generator, you need to have PHP 8.1+ installed and Composer.
 
 1. Move to the `generator` directory: `cd generator`
-1. Install dependencies: `composer install`
-1. Run the generator: `./generate`
+2. Install dependencies: `composer install`
+3. Run the generator: `./generate`
 
 ## Configuration
 
 The `generator/config/*.yaml` files contains the list of operators and stages that are supported by the library.
 
+### Arguments
+
+| Field | Type | Description |
+| `name` | `string` | The name of the argument. It can start with `$`. |
```

Comment:
I suppose a leading `$` is used when the parameter in the aggregation operator needs it, but it is not part of the name used in the generated factory? In that case, I would recommend specifying the last part, as it is important to know for people re-using these files.

Reply:
Added